.github/workflows/screenshots.yml

```diff
@@ -44,7 +44,7 @@ jobs:
         run: npm start --workspace app &
 
       - name: Send screenshots to Percy
-        run: npx percy exec --parallel -- npm run test:screenshots
+        run: npx percy exec --parallel -- npm run test:screenshots --workspace app
```

Comment:
With the tasks for `govuk-frontend` moving into `package/tasks` via https://github.com/alphagov/govuk-frontend/pull/3491 it's more appropriate for the "Review app" screenshots task to live inside the review app